### ISSUE TITLE
TypeError: Cannot read property 'mixins' of undefined

### ISSUE THREAD

**Issue by DylanGraham** [danmademe/express-vue-super-simple#3]
Hi, is there any fix for this?
Thanks

```
$ npm start

> express-vue-super-simple@1.0.0 start /home/dylan/WebstormProjects/express-vue-super-simple
> node index.js

Express server listening on port 3000
TypeError: Cannot read property 'mixins' of undefined
    at /home/dylan/WebstormProjects/express-vue-super-simple/node_modules/express-vue/dist/utils/render.js:115:78
    at renderUtil (/home/dylan/WebstormProjects/express-vue-super-simple/node_modules/express-vue/dist/utils/render.js:114:12)
    at renderHtmlUtil (/home/dylan/WebstormProjects/express-vue-super-simple/node_modules/express-vue/dist/utils/render.js:197:12)
    at renderComponents (/home/dylan/WebstormProjects/express-vue-super-simple/node_modules/express-vue/dist/utils/renderComponents.js:20:32)
    at /home/dylan/WebstormProjects/express-vue-super-simple/node_modules/express-vue/dist/index.js:19:37
Error: TypeError: Cannot read property 'mixins' of undefined
    at renderError (/home/dylan/WebstormProjects/express-vue-super-simple/node_modules/express-vue/dist/utils/renderError.js:8:14)
    at /home/dylan/WebstormProjects/express-vue-super-simple/node_modules/express-vue/dist/utils/renderComponents.js:23:35
```

**Comment by danmademe** [comment]
Yes. There was a bug in the library, I have fixed it and will update this example.. thank you! 

this was fixed in express-vue v3.9.6

Pull this repo and npm install

**Comment by danmademe** [comment]
https://github.com/danmademe/express-vue/pull/55
